script/travis_setup

```diff
@@ -35,13 +42,6 @@ else
     echo jpegresize exists
 fi
 
-if [ ! -f /usr/local/bin/jpegexiforient ]; then
```

Comment:
Corresponds to [Commit ac4a453](https://github.com/MushroomObserver/mushroom-observer/commit/ac4a453) ("Removed script/jpegexiforient.c")